openstack_lease_it/lease_it/backend/TestConnection.py

```diff
@@ -85,7 +92,11 @@ def _flavors(self):
 
         :return: dict()
         """
-        response = cache.get('flavors')
+
+        if GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
```

Comment:
Remplacer par
if not GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
response = cache.get('instances')